framework/Source/Kirakira.swift

```diff
@@ -151,6 +151,9 @@ public class Kirakira: OperationGroup {
     public var frameRate: Float = 60 {
         didSet { sparklesEffect.frameRate = frameRate }
     }
+    public var time: Float = 0 {
+        didSet { sparklesEffect.time = time }
+    }
```

Comment:
nit: it's more like `seed` instead of time.

Reply:
Hi @yyjim 
Yeah I'm on your side that it's more like the seed in our case.
How about changing the property name for Kirakira, but keeping using `time` in `Sparkles`?
So that we still have a position to align the name with `CBVisualEffectBuiltIn`.

Reply:
It's for creating Kirakira animation originally, but used for changing the result in PicCollage.
I have no concern to change the naming

Reply:
either way is fine for me 😄